test/HowToWriteTestsUsingFileCheck.md

```diff
@@ -79,6 +79,9 @@ annotations from the example above one would write:
 
 * `CHECK: <pattern>`
   Scans the input until `PATTERN` is found. Fails if the pattern is not found.
+* `CHECK-NEXT: <pattern>`
+  Scans the input on the line immediately following the previous CHECK until
+  `PATTERN` is found. Fails if the pattern is not found on that line.
```

Comment:
Consider making separate PR for changes that are not relevant to the functionality of PR

Reply:
Gotcha, will do that next time